content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md

```diff
@@ -76,28 +76,28 @@ jobs:
     - uses: {% data reusables.actions.action-checkout %}
 
     - name: Azure Login
-      uses: azure/login@14a755a4e2fd6dff25794233def4f2cf3f866955
+      uses: azure/login@v1
```

Comment:
(here and elsewhere) Our style guide requires us to use the full SHA for third party actions. It's ok if you want to update the workflow to use a later version, but we'll need to specify the SHA that corresponds to the new verison.